examples/transfer-ether.js

```diff
@@ -0,0 +1,13 @@
+const bn = require('bn.js')
+
+const utils = require('./utils')
+
+async function execute() {
+  const { matic, network } = await utils.getMaticClient('testnet', 'v3')
+  matic.setWallet(utils.getPrivateKey())
+
+  const amount = new bn(10).pow(new bn(17)) // 0.1 token
+  return matic.transferEthers(process.env.FROM, amount, { from: process.env.FROM })
```

Comment:
Add a comment saying, `// performing a self-transfer` so it's clearer what's happening here.